main.go

```diff
@@ -588,6 +591,11 @@ func main() {
 				Usage:   "Fail instead of warn on validation errors.",
 				EnvVars: []string{"STRICT_VALIDATIONS"},
 			},
+			&cli.BoolFlag{
+				Name:    "debug",
+				Usage:   "enable debug output",
+				EnvVars: []string{"KAIROS_AGENT_DEBUG"},
+			},
```

Comment:
don't we need to add a similar block to `manual-install`, `reset` etc?

Reply:
nope, this is a global flag at the main command level, so its available to all commands